### ISSUE TITLE
Tank Insulation - add area to rad loss calculation, remove in-built conversion to therms

### ISSUE THREAD

**Issue by koay9f** [ORNL-AMO/MEASUR-Tools-Suite#372]
insulated_tank_reduction.cpp

lines 141 - 144

```   double conv_cond_loss = overall_coeff * tank_area * delta_t ;
    double rad_loss       = physics::us::kStefanBoltzmann * input.tank_emissivity * tank_area
                          * (std::pow(input.tank_temperature, 4) - std::pow(input.ambient_temperature, 4))
                          ; 
```

adding ` * tank_area` to line 142
removing `/1e5` from lines 141 and 144

line 146
   ``` double annual_heat_loss = (heat_loss * static_cast<double>(input.operating_hours))```

remove /10 from line 146

the results will now be in btu - need to change bindings/conversions and doxygen? (Desktop issue #8613: https://github.com/ORNL-AMO/AMO-Tools-Desktop/issues/8613)
will need to change the IR algorightm doc. notes in issue [#8484](https://github.com/ORNL-AMO/AMO-Tools-Desktop/issues/8484)

**Comment by gwestORNL** [comment]
Moving this back, need to resolve blocking sub-issue. Nick can either handle sub issue or hand off to Omar.
